homeassistant/components/toon/config_flow.py

```diff
@@ -67,6 +67,7 @@ async def async_step_agreement(
         self, user_input: dict[str, Any] = None
     ) -> FlowResult:
         """Select Toon agreement to add."""
+        assert self.agreements is not None
         if len(self.agreements) == 1:
```

Comment:
```suggestion
        if self.agreements and len(self.agreements) == 1:
```

Reply:
I think that similar to data, agreements can never be None so I've adjusted the type hint